wafer/schedule/admin.py

```diff
@@ -218,6 +218,134 @@ def validate_schedule():
     return errors
 
 
+# Useful filters for the admin forms
+
+class BaseDayFilter(admin.SimpleListFilter):
+    # Common logic for filtering on Slots and ScheduleItem.slots by Day
+    # We need to do this as a filter, since we can't use sorting since
+    # day is dynamic (either the model field or the previous_slot)
+    title = _('Day')
```

Comment:
Does each filter that inherits from this need a unique title? I'm just wondering how things will look in the admin interface?

Reply:
Slots and ScheduleItems are different AdminForms so currently the children aren't ever on the same form. In this case the common name isn't a problem and adds a measure of consistency.

Reply:
Ah. Woot.